script.js

```diff
@@ -0,0 +1,13 @@
+const hamb = document.querySelector('.hamb');
+const navLinks = document.querySelector('.nav-links');
+
+hamb.addEventListener('click', () => {
+  hamb.classList.toggle('active');
+  navLinks.classList.toggle('active');
+});
+
+document.querySelectorAll('.navlink').forEach((n) => n
+  .addEventListener('click', () => {
+    hamb.classList.remove('active');
+    navLinks.classList.remove('active');
+  }));
```

Comment:
- [ ] Upon clicking on the nav links I also noticed some of the links don't redirect to the appropriate section of the page specifically the "Portfolio" link redirects to the "About myself" section and the "About" link redirects to the "Contact" section on your implementation. So please make sure to fix this as it will confuse users and lead to a bad UX. 😞